schema/artist/index.js

```diff
@@ -486,7 +514,7 @@ const Artist: GraphQLFieldConfig<ArtistType, *> = {
       type: new GraphQLNonNull(GraphQLString),
     },
   },
-  resolve: (root, { id }, request, resolver) => {
+  resolve: (_root, { id }, _request, resolver) => {
```

Comment:
was just going to ask if that s was a type lol.

Reply:
🙊